MyProject/cls/Employee.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cls
+{
+    class Employee: Person
+    {
+        public string Position { get; private set; }
+        public int AccessLevel { get; private set; }
+
+        private double Salary;
```

Comment:
private fields names should be in the form like : "_someNameField"

Reply:
Done